.config/sway/config.d/50-openSUSE.conf

```diff
@@ -92,6 +92,7 @@ exec_always {
     gsettings set org.gnome.desktop.interface cursor-theme 'Adwaita'
     test -e $SWAYSOCK.wob || mkfifo $SWAYSOCK.wob
     tail -f $SWAYSOCK.wob | $wob
+    swaync --style /etc/sway/swaync/style.css --config /etc/sway/swaync/config.json
```

Comment:
Also for swaync I would rather deploy these files to their default folder and start the daemon using the systemd user service that is shipped with it. Said service is disabled by default, we could enable it when we install openSUSEway.

Reply:
@FilippoBonazziSUSE we can't use default locations - they would be overwritten by the swaync package after package update:
https://build.opensuse.org/package/view_file/X11:Wayland/SwayNotificationCenter/SwayNotificationCenter.spec?expand=1

service - sure :) I would say it would be the best way. I choose the easiest :)
> systemd user service
we can't do user services, those are under `/home`. We can provide system service, and ppl could enable it on user level.

As for `exec` and default way - true if we leave `exec` here it would be hard to customize. But true for everything else here - wob. If ppl don't like it or would like to use something else.

Reply:
services would be good though for both wob and swaync. Customizing services is easy - just stop and disable it.

Reply:
The swaync systemd user service can be found in `/usr/lib/systemd/user/swaync.service`

Reply:
If we can't use the default config file location though (now I understand why), how can we tell the systemd service for e.g. `swaync` where to pick up the openSUSEway configuration?

Reply:
only modifying swaync service file upstream, so it would accept parameters like `Environment` and `EnvironmentFile` to customize what is passed to the `Exec` . But there is also d-bus service.

So no idea for now